docs/docs/quick-start.md

```diff
@@ -18,6 +18,8 @@ Video hosted on [egghead.io][egghead].
 npm install -g gatsby-cli
 ```
 
+> _Note: see [info about using `npx`](#note-on-using-npx), as referenced in the above video._
```

Comment:
We need to reshoot the video really. @jlengstorf could you put this on your TODOs?

Reply:
Ok, sounds good to me! I'll back out that change so it doesn't hold up this PR.

Reply:
Removed. I could also take a stab at re-recording it, as a fellow Egghead screencaster :)

Reply:
💪